drivers/serial/Kconfig.ns16550

```diff
@@ -61,6 +61,15 @@ config UART_NS16550_VARIANT_NS16950
 	  This enables support for 128-bytes FIFO and automatic hardware flow control.
 endchoice
 
+config UART_NS16550_FIFO_SIZE
```

Comment:
why is this both Kconfig and dts? IMO this is IP's hardware property, so with dts is enough.